index.html

```diff
@@ -0,0 +1,138 @@
+            <span class="first-headline"
+              >Hey there. I’m Titus Kiplagat</span
+            > <br/>
+            <span class="second-headline"
+              >I’m a software developer</span
+            >
+          </h1>
+          <p class="text-headline">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <ul class="social-icons">
+            <li class="social-icon">
+              <a
+                href="https://github.com/Titus-Kiplagat"
+                class="social-icon-link"
+              >
+                <img
+                  src="./assets/social-media/github.svg"
+                  alt="twitter icon"
+                  class="social-icon-img btn-zoom"
+                />
+              </a>
+            </li>
+            <li class="social-icon">
+              <a
+                href="hhttps://www.linkedin.com/in/titus-kiplagat-kemboi-9a8848262/"
+                class="social-icon-link"
+              >
+                <img
+                  src="./assets/social-media/linkedin.svg"
+                  alt="linkedin icon"
+                  class="social-icon-img btn-zoom"
+                />
+              </a>
+            </li>
+            <li class="social-icon">
+              <a
+                href="#"
+                class="social-icon-link"
+              >
+                <img
+                  src="./assets/social-media/angellist.svg"
+                  alt="angellist icon"
+                  class="social-icon-img btn-zoom"
+                />
+              </a>
+            </li>
+            <li class="social-icon">
+              <a
+                href="#"
+                class="social-icon-link"
+              >
+                <img
+                  src="./assets/social-media/twitter.svg"
+                  alt="twitter icon"
+                  class="social-icon-img btn-zoom"
+                />
+              </a>
+            </li>
+            <li class="social-icon">
+              <a
+                href="#"
+                class="social-icon-link"
+              >
+                <img
+                  src="./assets/social-media/medium.svg"
+                  alt="medium icon"
+                  class="social-icon-img btn-zoom"
+                />
+              </a>
+            </li>
+          </ul>
+          <a
+            href="#work"
+            class="headline-scroll"
+          >
+            <img
+              src="./assets/Disabled.svg"
+              alt="headline scroll icon"
+              class="scroll-icon btn-zoom"
+            />
+          </a>
+        </div>
+        <div class="headline-background">
+          <img
+            src="./assets/Header-llustration-desktop.svg"
+            alt="background screen display on desktop"
+          />
```

Comment:
- [ ] The section has the proper background image (decorations are always backgrounded images, not img tags) 👆 👆 I think it would be better to use the css background-image instead of adding them with an img tag,use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.use CSS background-image.  for more info.. 🙏 this normally increases the performance as the background images are loaded asynchronously, which means that they don't block the rest of the page from loading. This can improve the performance of your website, especially on slow connections. 😎😎